Toolchain/CMake/ClangToolchain.txt.in

```diff
@@ -33,7 +33,8 @@ set(CMAKE_EXE_LINKER_FLAGS_INIT "-Wl,--hash-style=gnu,-z,relro,-z,now,-z,noexecs
 # FIXME: We could eliminate this setting by building LibC and support asm files (crti.o, crtn.o)
 #        in a separate build stage before the main build to ensure that LibC is available
 #        for the try_compile check for the main build.
-set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
```

Comment:
I think leaving this here commented out with another explanation for the new workaround would be better, so whoever actually takes their time to pick this up will have a little more context.

Reply:
Added a few lines to document why `set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)` is not a suitable workaround.